example/objc/main_darwin.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build example
```

Comment:
I think we no longer need `example` tag as we dropped the support of Go 1.16 and 1.17: https://github.com/hajimehoshi/ebiten/issues/1462#issuecomment-1229493555